migration/1709468854359-project_actual_matching_v6.ts

```diff
@@ -0,0 +1,102 @@
+                    p.id, 
+                    p.slug, 
+                    p.title,
+                    qr.id as "qfId",
+                    qr."minimumPassportScore" as "minimumPassportScore",
+                    STRING_AGG(distinct pa."networkId" || '-' || pa."address", ', ') AS "networkAddresses",
+                    COALESCE(SUM(d."valueUsd"), 0) AS "allUsdReceived", 
+                    COUNT(DISTINCT d."fromWalletAddress") AS "totalDonors"
+                FROM 
+                    public.donation AS d
+                    INNER JOIN project p ON p.id = d."projectId"
+                    INNER JOIN qf_round qr ON qr.id = d."qfRoundId"
+                    INNER JOIN project_address pa ON pa."projectId" = p.id AND pa."networkId" = ANY(qr."eligibleNetworks")
+                    LEFT JOIN project_fraud pf ON pf."projectId" = p.id AND pf."qfRoundId" = qr.id
+                WHERE
+                    pf.id IS NULL
+                GROUP BY 
+                    p.id,
+                    p.title, 
+                    p.slug,
+                    qr.id
+            ), 
+            DonationsAfterAnalysis AS (
+                SELECT 
+                    p2.id, 
+                    p2.slug, 
+                    p2.title,
+                    qr.id as "qfId",
+                    COALESCE(SUM(d2."valueUsd"), 0) AS "allUsdReceivedAfterSybilsAnalysis", 
+                    COUNT(DISTINCT d2."fromWalletAddress") AS "uniqueDonors",
+                    SUM(SQRT(d2."valueUsd")) AS "donationsSqrtRootSum",
+                    POWER(SUM(SQRT(d2."valueUsd")), 2) as "donationsSqrtRootSumSquared"
+                FROM 
+                    public.donation AS d2
+                    INNER JOIN project p2 ON p2.id = d2."projectId"
+                    INNER JOIN qf_round qr ON qr.id = d2."qfRoundId"
+                    INNER JOIN project_address pa ON pa."projectId" = p2.id AND pa."networkId" = ANY(qr."eligibleNetworks")
+                    LEFT JOIN "sybil" s ON s."userId" = d2."userId" AND s."qfRoundId" = qr.id
+                    LEFT JOIN project_fraud pf ON pf."projectId" = p2.id AND pf."qfRoundId" = qr.id
+                WHERE 
+                    p2."statusId" = 5 
+                    AND LOWER(d2."fromWalletAddress") NOT IN (
+                        SELECT DISTINCT LOWER(pa.address)
+                        FROM public.project_address pa
+                        JOIN public.project p3 ON p3.id = pa."projectId"
+                        WHERE p3."verified" = true
+                        AND p3."listed" = true
+                    )
+                    AND NOT EXISTS (
+                        SELECT 1
+                        FROM public.project_address pa
+                        JOIN public.project p4 ON p4.id = pa."projectId"
+                        WHERE LOWER(d2."fromWalletAddress") = LOWER(pa.address)
+                        AND p4."verified" = true
+                        AND p4."listed" = true
+                    )
```

Comment:
Both queries do the same, as they exclude fromWalletAddresses that are in the project_address table. The Not Exists is more efficient, so I think we can remove the not in.